docs/check-conformance-test-suite-docs.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function init_settings() {
+  TEST_SUITE_DOCUMENT="unchecked"
+
+  STABLE_VERSION="$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)"
+  echo "STABLE_VERSION: $STABLE_VERSION"
+  
+  RELEASE_VERSION="$(echo "$STABLE_VERSION" | awk -F '.' '{print $1 "." $2}').0"
+  echo "RELEASE_VERSION: $RELEASE_VERSION"
+  
+  KUBE_CONFORMANCE_FILENAME="KubeConformance-$(echo "$STABLE_VERSION" | awk -F '.' '{print $1 "." $2}' | sed 's/v//').md"
+  echo "KUBE_CONFORMANCE_FILENAME: $KUBE_CONFORMANCE_FILENAME"
+  
+  CONFORMANCE_BASE_URL="${CONFORMANCE_BASE_URL:-https://raw.githubusercontent.com/cncf/k8s-conformance/master/docs}"
+  echo "CONFORMANCE_BASE_URL: $CONFORMANCE_BASE_URL"
+  
+  MD_FILE_URL=${CONFORMANCE_BASE_URL}/${KUBE_CONFORMANCE_FILENAME}
+  echo "MD_FILE_URL: $MD_FILE_URL"
+  
+  RESPONSE_CODE=$(curl -s -o /dev/null "$MD_FILE_URL" --write-out "%{http_code}")
+  if [[ $RESPONSE_CODE ==  "404" ]]; then
+    TEST_SUITE_DOCUMENT="missing"
+  fi
+}
+
+function setup_k8s_repo() {
+  mkdir -p /tmp/go/src/k8s.io
+  cd "$_"
+  git clone --depth 1 --branch "${RELEASE_VERSION}" https://github.com/kubernetes/kubernetes.git
+  export KUBE_ROOT=/tmp/go/src/k8s.io/kubernetes
+  cd "${KUBE_ROOT}"
+  git log -1
+}
+
+function install_go() {
```

Comment:
if this is run in a development environment with an existing Go installation, will it affect the existing installation?